src/main/java/lotto/domain/factory/LottoMakeAble.java

```diff
@@ -0,0 +1,8 @@
+package lotto.domain.factory;
+
+import lotto.domain.Lotto;
+
+public interface LottoMakeAble {
```

Comment:
```suggestion
public interface LottoMakeable {
```

`@FunctionalInterface` 👍 
하지만 이름에 오타가 있네용 @_@

Reply:
처음에 `LottoMakeable`로 했었는데
스펠링 틀렸다는 초록 밑줄이 그어져서
`LottoMakeAble`로 바꿨었습니다..
현재 `LottoMakeable`로 바꿔도 초록 밑줄이 그어지는데
스펠링이 안틀렸는데 왜이러는 걸까요 ㅠㅠ

Reply:
IDE가 만능은 아니라 가끔 올바르지 않게 경고를 줄 때가 있습니다 ㅜㅜ

Reply:
아하 그렇군요...!